app/src/main/java/org/wikipedia/suggestededits/SuggestedEditsTasksFragment.kt

```diff
@@ -388,6 +391,27 @@ class SuggestedEditsTasksFragment : Fragment() {
         }
     }
 
+    private fun showSurveyDialog() {
```

Comment:
I have another idea: how about we move this survey dialog into `ContributionsDashboardHelper` so that it will be easier to manage and remove in the future?